tests/user_friendly/test_uf_utils.py

```diff
@@ -0,0 +1,857 @@
+import math
+
+import fiftyone as fo
+import motmetrics as mm
+import numpy as np
+import pandas as pd
+import pytest
+
+from seametrics.payload import Payload, Resolution, Sequence
+from seametrics.user_friendly.utils import (
+    build_metrics_template,
+    calculate,
+    calculate_from_payload,
+    num_gt_ids,
+    realize_metrics,
+    recognition,
+    sum_dicts,
```

Comment:
I have mixed feeling with this, on one hand this ensures we have full coverage, but... makes tests quite hard to maintain if we need a refactor.

For upcoming unit tests, I would only focus on testing the metric as a whole instead of the detailed functions/methods used inside the metric.